packages/shared/src/modules/contributors/libs/validation-schemas/contributor-patch.validation-schema.ts

```diff
@@ -8,6 +8,7 @@ import { type ContributorPatchRequestDto } from "../types/types.js";
 
 const contributorPatch: z.ZodType<ContributorPatchRequestDto> = z
 	.object({
+		hiddenAt: z.union([z.string().nullable(), z.null()]),
```

Comment:
Can't we do it like this?
```suggestion
		hiddenAt: z.string().nullable(),
```

Reply:
or at least
```suggestion
		hiddenAt: z.union([z.string(), z.null()]),
```